definitions/messages-olympus.yml

```
@@ -1,6 +1,6 @@
 openapi: "3.0.1"
 info:
-  version: 1.2.2
+  version: 1.2.1
```

```suggestion
  version: 1.2.3
```

Not sure why we're decrementing the patch version here. We probably want to increment it.